rq/scheduler.py

```diff
@@ -161,8 +161,12 @@ def enqueue_scheduled_jobs(self):
                         try:
                             queue._enqueue_job(job, pipeline=pipeline, at_front=bool(job.enqueue_at_front))
                         except DeserializationError:
-                            self.log.error('Deserialization error while trying to enqueueing job %d', job.id, exc_info=True, stack_info=True)
-                        registry.remove(job, pipeline=pipeline)
+                            self.log.error('RQ Scheduler Error: deserialization error while trying to enqueueing job %d', job.id, exc_info=True, stack_info=True)
```

Comment:
```suggestion
                            self.log.error('RQ Scheduler Error: deserialization error while trying to enqueue job %d', job.id, exc_info=True, stack_info=True)
```

Feel free to ignore, just a grammar error.

Reply:
thanks! 😅